Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gableroux/flutter:v1.7.6
+
+WORKDIR /app
+
+COPY ./app_feup/ .
```

Comment:
This will move all the files inside the `app_feup` directory to `/app` - they will not be in `/app/app_feup` but instead in `/app`. Is this intended?

Reply:
Yes that is the intended behavior